src/components/image-description-block.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+import {
+  Image,
+  ImageProps,
+  StyleProp,
+  StyleSheet,
+  TouchableOpacity,
+  View,
+  ViewStyle,
+} from 'react-native';
+
+import {Color} from '@app/colors';
+import {createTheme} from '@app/helpers';
+import {I18N} from '@app/i18n';
+
+import {Spacer, Text, TextProps} from './ui';
+
+export interface ImageDescriptionBlockProps {
+  source: ImageProps['source'];
+  style?: StyleProp<ViewStyle>;
+  title: I18N;
+  titleParams?: TextProps['i18params'];
+  description: I18N;
+  descriptionParams?: TextProps['i18params'];
+  onPress?(): void;
+}
+
+export const ImageDescriptionBlock = ({
+  source,
+  style,
+  title,
+  description,
+  titleParams,
+  descriptionParams,
+  onPress,
+}: ImageDescriptionBlockProps) => {
+  return (
+    <View style={StyleSheet.flatten([styles.container, style])}>
```

Comment:
https://reactnative.dev/docs/stylesheet#flatten

Flatten will be recreated every render, so you need to memoize it
```suggestion
    <View style={StyleSheet.flatten([styles.container, style])}>
```